.github/actions/generate-system-logs/action.yml

```diff
@@ -0,0 +1,32 @@
+name: ""
+description: "Generate system logs for hardware debugging"
+
+inputs:
+  run_args:
+    description: 'Commands to run in docker'
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - name: Get Runner Host Name
+      id: get_hostname
+      shell: bash
+      run: |
+        echo "HOSTNAME=$(hostname)" >> $GITHUB_ENV
+        echo "TIMESTAMP=$(date +'%Y%m%d%H%M%S')" >> $GITHUB_ENV
+    - name: Determine Docker Tag to use
+      shell: bash
+      run: |
+        mkdir -p ~/run-log/
+        sudo dmesg > ~/run-log/dmesg.log
+        sudo lspci > ~/run-log/lspci.log
+        sudo lshw > ~/run-log/lshw.log
+        tt-smi-metal -s -f ~/run-log/tt_smi_${{ env.TIMESTAMP}}_${{ env.HOSTNAME }}_dump.log
```

Comment:
the process isn't killed by GH when a job times out. You shouldn't be calling `tt-smi -s` when a process is still alive

Reply:
fair point. I didn't consider that case.
I'll remove that the tt-smi dump, and if users are interested in that data, they can refer to the reset step which shows on the log. 

I'm not sure how clean it will be to get the PID of the process esp if it becomes multi-process